STANDARDS.md

```diff
@@ -116,13 +119,13 @@ Pre-submission filtering:
 Post-submission filtering:
```

Comment:
Given the standards above for versions, doing any post-submission filtering would require a minor version bump. Is that how we intend this to be used? That would mean that any released dataset would be v3.1, and nothing would ever be v3.0 unless all the stereochemistry, topology, etc was perfect and didn't need to be filtered.

Reply:
Trevor mentioned that it's referring to the [`UnperceivableStereoFilter`](https://docs.openforcefield.org/projects/qcsubmit/en/stable/_modules/openff/qcsubmit/results/filters.html#UnperceivableStereoFilter), which writes the molecule to an SDF file and tries to read it back in. I think this has to be applied after an optimization, since the QM structure and the structure from the conformer generation may differ. But for a single point dataset, it could be done before.

I updated the text to mention that, and specify which other filters can be used.

Reply:
Thanks @amcisaac!